### PR TITLE
Added expam package

### PR DESCRIPTION
**Pull request by seansolari**
Add expam package to bioconda.

Expam is a Python-based package for phylogenetic analysis of metagenomes.

### CODE REVIEW

**Comment by seansolari** [comment]
@BiocondaBot please add label

File: recipes/expam/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - numpy ==1.22.2
+    - setuptools
+  run:
+    - python
+    - numpy ==1.22.2
+    - matplotlib-base ==3.5.1
+    - pandas ==1.4.1
+    - psutil ==5.9.0
+    - requests ==2.27.1
+    - pytables ==3.7.0
```
Comment (dpryan79) [inline recipes/expam/meta.yaml:33]: Does it actually require these exact versions? That will make it almost immediately incompatible with other packages.

Comment (seansolari) [reply recipes/expam/meta.yaml]: Thanks for that @dpryan79, I modified the requirements and reduced it down to something less restricting. The new version seems to pass all the tests (c6c15d1 below).

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**